rel-eng/uyuni-check-version

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python3
+
+from argparse import ArgumentParser
+from configparser import ConfigParser
+from itertools import chain
+from os.path import dirname, expanduser, realpath
+from re import match
+from sys import exit
+import urllib.parse
+import urllib.request
+
+PRJ = 'systemsmanagement:Uyuni:Master'
+PACKAGES = ['patterns-uyuni', 'uyuni-docs_en',
+            'release-notes-uyuni', 'release-notes-uyuni-proxy']
+
+
+def webui_version(conf='%s/../web/conf/rhn_web.conf' % dirname(realpath(__file__))):
+    config = ConfigParser()
+    with open(conf) as f:
+        # This is basically a hack, as our .conf file does not have sections
+        config.read_file(chain(['[section]'], f), source=conf)
+        return config.get('section', 'web.version.uyuni')
+
+
+def obs_get_package_ver(args, project, package):
+    url = "%s/source/%s/%s/%s.spec" % (args.apiurl, project, package, package)
+    user = args.user
+    password = args.password
+    password_mgr = urllib.request.HTTPPasswordMgrWithDefaultRealm()
+    password_mgr.add_password(None, url, user, password)
+    auth_handler = urllib.request.HTTPBasicAuthHandler(password_mgr)
+    opener = urllib.request.build_opener(auth_handler)
+    urllib.request.install_opener(opener)
+    req = urllib.request.Request(url=url, method='GET')
+    resource = urllib.request.urlopen(req)
+    charset = resource.headers.get_content_charset()
+    if charset is None:
+        charset = 'utf-8'
+    for line in resource.read().decode(charset).split('\n'):
+        version = match('^Version:\s*(\S+)$', line)
+        if version:
+            return version.group(1)
+
+
+def parse_arguments():
+    """ Parse arguments from command line """
+    parser = ArgumentParser(
+        description="Check if Uyuni versions are aligned at all packages before a release")
+    parser.add_argument("-u", "--user", action="store", dest="user",
+                        help="OBS Username or read from ~/.oscrc")
+    parser.add_argument("-p", "--password", action="store", dest="password",
+                        help="OBS Password or read from ~/.oscrc")
+    parser.add_argument("-a", "--api-url", action="store", dest="apiurl",
+                        default="https://api.opensuse.org",
+                        help="OBS API URL (Default: https://api.opensuse.org")
+    args = parser.parse_args()
+    if not args.user or not args.password:
+        try:
+            creds_path = "%s/.oscrc" % expanduser('~')
```

Comment:
IIRC new osc versions have a new path for this file, so maybe we will need to check both.